vehicle/polestar/identity.go

```diff
@@ -96,6 +96,16 @@ func (v *Identity) login() (*oauth2.Token, error) {
 		return nil, err
 	}
 
+	// If the authorization code is empty, this indicates that user consent must be handled
+	// before the code can be obtained. The `confirmConsentAndGetCode` method is called as a
+	// workaround to guide the user through the consent process and retrieve the authorization code.
+	if code == "" {
```

Comment:
We will never reach this as in line 96 we will return due to the error of not having the code.

Reply:
Damn it – I missed the early return!

Reply:
lgtm